jansi/src/main/java/org/fusesource/jansi/AnsiConsole.java

```diff
@@ -100,6 +104,11 @@ public void close() throws IOException {
 		};
 	}
 
+	private static boolean isCygwin(){
+		String term = System.getenv("TERM");
+		return (term != null && term.equals("cygwin"));
```

Comment:
I also believe this should say "xterm", which is what `TERM` is set to in MSYS(2) / Git for Windows, too. I'll post a follow-up PR which fixes this.